ADR/0041-integration-service-upstreaming.md

```diff
@@ -0,0 +1,95 @@
+# XX. Creating Artifacts from Snapshots for testing and release
+
+Date: 2024-10-21
+
+## Status
+
+Proposed
+
+## Authors
+
+- Krunoslav Pavic
+- Andrew McNamara
+
+## Context
+
+This ADR aims to cover the proposed functionality for creating additional Artifacts as part of the Konflux build/test/release process.
+These artifacts would vary in type and would cover use cases where either the single built image or entire sets of images
+from the user's application are needed to generate one or more artifacts.
+
+## Use Cases in Detail
+
+* A team produces operands and operators. When integration service creates a Snapshot, it uses a constructor that
+  actually builds a bundle. When releasing, the team releases that single bundle artifact. That constructor also has an extract
```

Comment:
| it uses a constructor that actually builds a bundle

what does it mean? which constructor?

Reply:
nvm, I see an explanation about it below https://github.com/konflux-ci/architecture/pull/212/files#diff-d6bbded950ff5bcfe768192ca004473b0d89e927f5f8cd128922a6dd42b5a167R44